DeepSpeech.py

```diff
@@ -358,8 +358,8 @@ def BiRNN(batch_x, seq_length, dropout):
     # clipped RELU activation and dropout.
 
     # 1st layer
-    b1 = variable_on_worker_level('b1', [n_hidden_1], tf.random_normal_initializer(stddev=FLAGS.b1_stddev))
-    h1 = variable_on_worker_level('h1', [n_input + 2*n_input*n_context, n_hidden_1], tf.random_normal_initializer(stddev=FLAGS.h1_stddev))
+    b1 = variable_on_worker_level('b1', [n_hidden_1], tf.contrib.layers.xavier_initializer(uniform=False))
+    h1 = variable_on_worker_level('h1', [n_input + 2*n_input*n_context, n_hidden_1], tf.contrib.layers.xavier_initializer(uniform=False))
```

Comment:
Issue #319 mentioned only first and last layers, but why not using this adaptive approach also for the other layers?
Something to check: Somewhere I read about ReLUs requiring a higher input variance than usual non-linearities, as half of their activation is 0.